docs/x509.rst

```diff
@@ -718,6 +754,17 @@ X.509 Extensions
 
         :returns: A list of values extracted from the matched general names.
 
+        .. doctest::
+
+            >>> from cryptography import x509
+            >>> from cryptography.hazmat.backends import default_backend
+            >>> from cryptography.hazmat.primitives import hashes
+            >>> cert = x509.load_pem_x509_certificate(cryptography_cert_pem, default_backend())
+            >>> ext = cert.extensions.get_extension_for_oid(x509.OID_SUBJECT_ALTERNATIVE_NAME)
+            >>> san = ext.value
```

Comment:
I don't think this needs it's own local var, it makes stuff look more verbose than it is